### PR TITLE
Migration Material UI V5

### PR DESCRIPTION
**Pull request by phibersoft**
Typescript example has been upgraded from v4 to v5 in accordance with the Material UI Official documentation.

### CODE REVIEW

**Review by andirsun — APPROVED** [review]
Looks good but I will wait for another Mantainer approval.

**Review by saltyshiomix — APPROVED** [review]
@phibersoft 

Sorry for late reply and thank you for your PR 🥇 

LGTM and it works perfectly :)

**Pull request merged**
